lib/puppet/application/device.rb

```diff
@@ -77,10 +77,10 @@ def help
 
 USAGE
 -----
-  puppet device [-d|--debug] [--detailed-exitcodes] [-V|--version]
+  puppet device [-d|--debug] [--detailed-exitcodes] [--deviceconfig <file>]
```

Comment:
Since deviceconfig takes an optional parameter, can you update the usage to show that?